package br.unisinos.library.dto;

import jakarta.validation.constraints.NotBlank;

public record CategoriaRequestDTO (

    @NotBlank
    String nome
    
) {}